package.json

```diff
@@ -272,17 +272,17 @@
         {
           "id": "snyk.views.analysis.code.quality",
           "name": "Code Quality",
-          "when": "snyk:lsCodePreview && snyk:loggedIn && snyk:featuresSelected && snyk:codeEnabled && !snyk:codeLocalEngineEnabled && snyk:workspaceFound && !snyk:error"
```

Comment:
We should keep !codeLocalEngineEnabled context here and above, as we don't want to show results view, when LCE is enabled